REQUIRE

```diff
@@ -1,5 +1,4 @@
 julia 0.4.6
-JSON 0.8.0
+JSON 0.6.0
 MathProgBase 0.5.4
 JuMP 0.14.0
-Compat 0.9.3
```

Comment:
Good catch.  I still have much to learn about this.

The core dependency in the code appears to be `readall -> readstring`, hence min version should be [compat v0.7.9](https://github.com/JuliaLang/Compat.jl/commit/8468afb0c11e1c51e2217913cc635b077f3064e6).